src/pudl_rmi/formatter_optimus.py

```diff
@@ -35,7 +35,7 @@
     "remaining_life_avg": "Current Remaining Accounting Life (Yrs)",
     "plant_balance_w_common": "Gross Plant Balance/Original Cost ($)",
     "book_reserve_w_common": "Book Reserve/Accumulated Depreciation ($)",
-    "unaccrued_balance_w_common": "Current Net Plant Balance ($)",
```

Comment:
Do you need to create a new definition for unaccrued balance or are you not including it in the optimus format at all

Reply:
I'm not including it. Turns out they really just wanted npb